### ISSUE TITLE
Recent Commit is Crashing Build

### ISSUE THREAD

**Issue by mattdesl** [browserify/module-deps#144]
To reproduce, install latest browserify (16.1.1) and run it on the following:

```js
// --> need a require(), or a commented out require() somewhere
// require('foo-bar');
var foo = 1;
const foo = 2;
```

```
browserify index.js
```

Resulting error comes from `module-deps`:

```
Error: Parsing file /Users/matt/Desktop/foobar/index.js: Identifier 'foo' has already been declared (4:6)
    at Deps.parseDeps (/Users/matt/Desktop/foobar/node_modules/module-deps/index.js:506:28)
    at getDeps (/Users/matt/Desktop/foobar/node_modules/module-deps/index.js:436:44)
    at /Users/matt/Desktop/foobar/node_modules/module-deps/index.js:420:32
    at ConcatStream.<anonymous> (/Users/matt/Desktop/foobar/node_modules/concat-stream/index.js:37:43)
    at emitNone (events.js:111:20)
    at ConcatStream.emit (events.js:208:7)
    at finishMaybe (/Users/matt/Desktop/foobar/node_modules/readable-stream/lib/_stream_writable.js:620:14)
    at endWritable (/Users/matt/Desktop/foobar/node_modules/readable-stream/lib/_stream_writable.js:628:3)
    at ConcatStream.Writable.end (/Users/matt/Desktop/foobar/node_modules/readable-stream/lib/_stream_writable.js:584:41)
    at DuplexWrapper.onend (/Users/matt/Desktop/foobar/node_modules/readable-stream/lib/_stream_readable.js:577:10)
```

**Comment by substack** [comment]
I think this was fixed by #145.

**Comment by substack** [comment]
I think maybe this is still an issue. After a syntax error budo isn't resuming watching for files changes.
